backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/api/ILinksDescriptionService.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.spring.collaborative.forms.api;
+
+import java.util.List;
+
+import org.eclipse.sirius.web.forms.description.FormDescription;
+
+/**
+ * This class gives access to all the links descriptions available.
+ *
+ * @author ldelaigue
+ */
+public interface ILinksDescriptionService {
```

Comment:
I don't understand why. I would understand that this service should be in sirius-web rather than sirius-components. But why should we limit link providers to only one?